components/onboarding/OnboardingVisualStepper.tsx

```diff
@@ -0,0 +1,64 @@
+import StatusComplete from '../shared/icons/StatusComplete';
+import StatusCurrent from '../shared/icons/StatusCurrent';
+import StatusIncomplete from '../shared/icons/StatusIncomplete';
+
+const OnboardingVisualStepper = ({ step }: { step: number }) => {
+  const renderSteps = () => {
+    const RepeatedPatternIncomplete = (number: number) =>
+      Array(number).fill(
+        <>
+          <div className="border border-black-600 w-28"></div>
+          <StatusIncomplete />
+        </>
+      );
+
+    const RepeatedPatternComplete = (number: number) =>
+      Array(number).fill(
+        <>
+          <StatusComplete />
+          <div className="border border-primary-500 w-28"></div>
+        </>
+      );
```

Comment:
I tried to extract these to eliminate duplicate code, but I am getting a react warning:
<img width="401" alt="Screenshot 2024-03-22 at 12 02 53 AM" src="https://github.com/DDVVPP/GitNote/assets/47732125/4e5145b8-7ab4-4c40-ab32-fb0f3d905977">

If I wrap a div around them with a key, then it messes up the styling...